src/environments/array.js

```diff
@@ -786,7 +786,31 @@ defineEnvironment({
         const symNode = checkSymbolNodeType(args[0]);
         const colalign: AnyParseNode[] =
             symNode ? [args[0]] : assertNodeType(args[0], "ordgroup").body;
-        const cols = colalign.map(function(nde) {
+        let myColAlign = [];
+        if (colalign[0].text === "*") {
+            const repeatCol = assertNodeType(colalign[1], "ordgroup");
```

Comment:
This seems to only support `*{123}` with braces. But for single-character numbers, you can omit the braces. For example, `*5` should work.